money/money.go

```diff
@@ -25,3 +29,11 @@ type Money struct {
 func NewMoney(amount float64, currency Currency) Money {
 	return Money{uint64(amount * 100000000), currency}
 }
+
+func (value *Money) String() string {
```

Comment:
Lint won't be happy with this - do comment on the exported method

Reply:
Fixed